### PR TITLE
Combining deposit and withdrawal services

### PR DESCRIPTION
**Pull request by dolgopolovwork**
### Description of the Change
Now we can run deposit and withdrawal services simultaneously using `./gradlew runBtcDepositWithdrawal` But you are still able to run them separately using `./gradlew runBtcNotary` and `./gradlew runBtcWithdrawal` respectively.

### CODE REVIEW

File: configs/btc/dw-bridge_local.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Iroha peer hostname
+btc-dw-bridge.iroha.hostname=d3-iroha
+# Iroha peer port
+btc-dw-bridge.iroha.port=50051
+# --------- Bitcoin ---------
+btc-dw-bridge.bitcoin.walletPath=deploy/bitcoin/regtest/d3.wallet
+btc-dw-bridge.bitcoin.blockStoragePath=deploy/bitcoin/regtest/blocks/dw-bridge
+btc-dw-bridge.bitcoin.confidenceLevel=6
+btc-dw-bridge.bitcoin.hosts=d3-btc-node0
+btc-dw-bridge.healthCheckPort=7074
```
Comment (grimadas) [inline configs/btc/dw-bridge_local.properties]: Add new line

File: configs/btc/dw-bridge_mainnet.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Iroha peer hostname
+btc-dw-bridge.iroha.hostname=d3-iroha
+# Iroha peer port
+btc-dw-bridge.iroha.port=50051
+# --------- Bitcoin ---------
+btc-dw-bridge.bitcoin.walletPath=deploy/bitcoin/mainnet/d3.wallet
+btc-dw-bridge.bitcoin.blockStoragePath=deploy/bitcoin/mainnet/blocks/dw-bridge
+btc-dw-bridge.bitcoin.confidenceLevel=6
+btc-dw-bridge.bitcoin.hosts=d3-btc-node0
+btc-dw-bridge.healthCheckPort=7074
```
Comment (grimadas) [inline configs/btc/dw-bridge_mainnet.properties]: NL

File: configs/btc/dw-bridge_testnet.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# Iroha peer hostname
+btc-dw-bridge.iroha.hostname=d3-iroha
+# Iroha peer port
+btc-dw-bridge.iroha.port=50051
+# --------- Bitcoin ---------
+btc-dw-bridge.bitcoin.walletPath=deploy/bitcoin/testnet/d3.wallet
+btc-dw-bridge.bitcoin.blockStoragePath=deploy/bitcoin/testnet/blocks/dw-bridge
+btc-dw-bridge.bitcoin.confidenceLevel=0
+btc-dw-bridge.bitcoin.hosts=d3-btc-node0
+btc-dw-bridge.healthCheckPort=7074
```
Comment (grimadas) [inline configs/btc/dw-bridge_testnet.properties]: NL

File: btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeAppConfiguration.kt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            irohaNetwork(),
+            withdrawalConfig.mstRegistrationAccount,
+            withdrawalConfig.changeAddressesStorageAccount
+        )
+    }
+
+    @Bean
+    fun blockStoragePath() = dwBridgeConfig.bitcoin.blockStoragePath
+
+}
```
Comment (Alexey-N-Chernyshov) [inline btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeAppConfiguration.kt]: please, set "Ensure line feed at file and on Save" in Settings->editor->general

File: btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeAppConfiguration.kt [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+import withdrawal.btc.provider.BtcWhiteListProvider
+import withdrawal.btc.statistics.WithdrawalStatistics
+import java.io.File
+
+val withdrawalConfig =
+    loadConfigs("btc-withdrawal", BtcWithdrawalConfig::class.java, "/btc/withdrawal.properties").get()
+val notaryConfig = loadConfigs("btc-notary", BtcNotaryConfig::class.java, "/btc/notary.properties").get()
+val dwBridgeConfig = loadConfigs("btc-dw-bridge", BtcDWBridgeConfig::class.java, "/btc/dw-bridge.properties").get()
+
+@Component
```
Comment (Alexey-N-Chernyshov) [inline btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeAppConfiguration.kt]: Please, add comments describing the component.

File: btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeConfig.kt [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+import config.IrohaConfig
+
+interface BtcDWBridgeConfig {
+
+    val bitcoin: BitcoinConfig
+
+    val iroha: IrohaConfig
+
+    val healthCheckPort: Int
+}
```
Comment (Alexey-N-Chernyshov) [inline btc-dw-bridge/src/main/kotlin/dwbridge/btc/config/BtcDWBridgeConfig.kt]: sap

File: btc-dw-bridge/src/main/kotlin/dwbridge/btc/main.kt [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+            logger.error("Cannot run btc deposit/withdrawal bridge", ex)
+            System.exit(1)
+        }
+}
+
+private fun webPortProperties(): Map<String, String> {
+    val properties = HashMap<String, String>()
+    properties["server.port"] = dwBridgeConfig.healthCheckPort.toString()
+    return properties
+}
```
Comment (Alexey-N-Chernyshov) [inline btc-dw-bridge/src/main/kotlin/dwbridge/btc/main.kt]: sap

**Pull request merged**